src/DotNetWorker/DotNetWorker.csproj

```diff
@@ -27,4 +27,10 @@
   <ItemGroup>
      <Protobuf Include=".\protobuf\src\proto\**\*.proto" ProtoRoot=".\protobuf\src\proto" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Protobuf Update=".\protobuf\src\proto\FunctionRpc.proto" Access="Internal" />
```

Comment:
If we use `GrpcServices="Client"` does that get us less code-gen? Looks like by default it generates Server and Client...

Reply:
I debated changing, but didn't want to increase the scope of the changes. Figured we could have this as a follow up change after some additional validation.